modules/data_release/jsx/managePermissionsForm.js

```diff
@@ -97,9 +95,14 @@ class ManagePermissionsForm extends Component {
                     }
                   /><br/>
                 </div>
-              )}
-            />
-          )}
+            );
+
+            return <StaticElement
+                      key={userId}
+                      label={user.name}
+                      text={<div>{versions}</div>}
```

Comment:
I think the div here is not necessary?

Reply:
I can't remember why but I'm pretty sure it was to fix a warning

Reply:
Oh, ok then :)